embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/getter/FloatColumnGetter.java

```diff
@@ -43,7 +42,15 @@ public void longColumn(Column column)
         long l;
         try {
             // TODO configurable rounding mode
-            l = DoubleMath.roundToLong(value, RoundingMode.HALF_UP);
+            if (Math.getExponent(value) > Double.MAX_EXPONENT) {
```

Comment:
this look duplicated with https://github.com/embulk/embulk-input-jdbc/pull/207/files#diff-4dda1f846176bb53b2e92ac04a515c83f15193e310c295087a62fb7c515e5586R45, could we avoid it?

Reply:
Hmm, let me consider having this as a method in `AbstractColumnGetter`.

Reply:
Refactored out in https://github.com/embulk/embulk-input-jdbc/pull/207/commits/bc8ebcb911f6dd3d817521b04142548f5fe5d228#diff-a75524cd471aa6334d9b7362f0578902b884cc7ffd65bd9e7872e1b3812a8a9eR106-R130

But, they had a difference -- one receiving `float`, and the other receiving `double`. I thought about merging them to `double`, but kept as-is in `AbstractColumnGetter` fow now so that it does not introduce unnecessary impacts.